src/styles/blocks/product.scss

```diff
@@ -0,0 +1,21 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &__photo {
+    width: 100%;
+    border-radius: 4px;
+  }
+
+  &__price {
+    bottom: 0;
+  }
+
+  &__category {
+    margin-top: 0;
```

Comment:
The 'margin-top: 0;' property for the '.product__category' class might not be necessary. If you want to ensure no default margin is applied, use 'margin: 0;' instead.